.gitlab-ci.yml

```diff
@@ -65,8 +65,7 @@ stages: # ---------------------------------------------------------------------
   script:
     - cd IntegrationTests/${PROJ_NAME}/script
     - pwd; ls -la; #debug
-    - vivado_hls -f ./compileHLS.tcl
-    - vivado -mode batch -source ./makeProject.tcl | awk 'BEGIN{IGNORECASE=1} /^error/ {exit_code=1;} // {print $0;} END{exit exit_code}'
+    - make -j4 Work
```

Comment:
The CMSSW scram command lets one specify "-j4". But it also lets one specify "-j" meaning "use all available cores". Can you provide this option? On linux, the number of available cores can be determined with the "nproc" command.

Reply:
According to the man page for scram and gmake, "-j" without an argument does not limit the number of processes that run simultaneously. So it's essentially equivalent to something like "-j999".

But I think we want to limit it to the number of CPU cores, so I now use "nproc" as you recommend.

Reply:
is this resolved, @tomalin ?